packages/textlint-config/readme.md

```diff
@@ -0,0 +1,31 @@
+# @strv/textlint-config
```

Comment:
It would be better if the title and the directory name matched the published package's name for easy navigation.